src/Dispatcher/Masa.Contrib.Dispatcher.IntegrationEvents.EventLogs.EF/Internal/EfCommon.cs

```diff
@@ -0,0 +1,45 @@
+namespace Masa.Contrib.Dispatcher.IntegrationEvents.EventLogs.EF.Internal;
+
+internal static class EfCommon
```

Comment:
EfCommon is a bad name. Maybe `TableStructureCache` or `DbContextExtensions`

Reply:
processed